data/json/items/generic/toys_and_sports.json

```diff
@@ -244,5 +244,65 @@
     "volume": "250 ml",
     "bashing": 7,
     "to_hit": 2
+  },
+  {
+    "type": "GENERIC",
+    "id": "mannequin",
+    "symbol": "@",
+    "color": "dark_gray",
+    "name": { "str": "mannequin" },
+    "category": "other",
+    "description": "A figure of an adult human, as seen in the display case at the mall. I t might be able to fool some zombies if it could attract their attention.",
```

Comment:
```suggestion
    "description": "A figure of an adult human, as seen in the display case at the mall.  It might be able to fool some zombies if it could attract their attention.",
```